src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/WellKnownDocumentationUrls.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DotNet.UpgradeAssistant
+{
+    public static class WellKnownDocumentationUrls
+    {
+        public static readonly Uri UpgradeAssistantUsageDocumentationLink = new Uri("https://github.com/dotnet/upgrade-assistant#usage");
```

Comment:
If this is in `DocumentationUrls` class, probably don't need `DocumentationLink` at the end of the name
```suggestion
        public static readonly Uri UpgradeAssistantUsage = new Uri("https://github.com/dotnet/upgrade-assistant#usage");
```